core/common/src/main/java/alluxio/conf/InstancedConfiguration.java

```diff
@@ -304,14 +304,12 @@ public boolean getBoolean(PropertyKey key) {
     Preconditions.checkArgument(delimiter != null,
         "Illegal separator for Alluxio properties as list");
     String rawValue = get(key);
-
     return ConfigurationUtils.parseAsList(rawValue, delimiter);
   }
 
   @Override
   public <T extends Enum<T>> T getEnum(PropertyKey key, Class<T> enumType) {
-    String rawValue = get(key);
-
+    String rawValue = get(key).toUpperCase();
```

Comment:
Not sure is there some corner case that the member of T enumType is lowerCase or mix lowerCase and upperCase?

Reply:
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

enumType should be in upper case as they are constant to follow Java convention. 
Also in here,  it only works for a few enum type propertyKey. I believe they are all in upper case

Reply:
OK, sounds good.